src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -147,16 +147,19 @@ function PIASubHeader({
 
           <ul className="dropdown-menu">
             <li role="button">
-              <a className="dropdown-item" onClick={() => handleDownload()}>
+              <button
+                className="dropdown-item"
+                onClick={() => handleDownload()}
+              >
                 Download
-              </a>
+              </button>
             </li>
             <li role="button">
               {/* Save or Edit button */}
               {!nextStepAction && mode === 'edit' && (
-                <a onClick={() => onSaveChangeClick} className="dropdown-item">
+                <button onClick={onSaveChangeClick} className="dropdown-item">
                   {secondaryButtonText}
-                </a>
+                </button>
```

Comment:
same as above

Reply:
do you mean in the /pia/new/intake we do not allow users to click the save button? I think we should allow user click the save button at any time.

Reply:
makes sense. Dummy save button then